File: redsheet-portal/src/app/database-models/s-m-t-p-setting.ts
import { required, maxLength, range, nested } from '@rx/annotations';
import {  vSMTPSettingRecord  } from './'
export class SMTPSetting {
    constructor(sMTPSetting?: SMTPSetting  | vSMTPSettingRecord )  {
        let properties = [ "isActive", "isSSL", "sMTPFromEmailAddress", "sMTPHostName", "sMTPPassword", "sMTPPort", "sMTPSettingsId", "sMTPUserName", "statusId",];
        for (let property of properties)
            if (sMTPSetting && sMTPSetting[property])
                this[property] = sMTPSetting[property];
    }
 
	isActive : boolean = false ;
 
	isSSL : boolean = false ;
 
    @maxLength(100)
	sMTPFromEmailAddress : string =   undefined;
 
    @maxLength(100)
	sMTPHostName : string =   undefined;
 
    @maxLength(100)
	sMTPPassword : string =   undefined;
 
	sMTPPort : number =   undefined;
 
	sMTPSettingsId : number =   0 ;
 
    @maxLength(100)
	sMTPUserName : string =   undefined;
 
	statusId : number =   undefined;


}

File: RedSheet.DbEntities/Models/SMTPSetting.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("SMTPSettings", Schema = "dbo")]
	[Serializable]
	public partial class SMTPSetting  
	{

	public Nullable<bool> IsActive { get; set; } 

	public Nullable<bool> IsSSL { get; set; } 
 
	[MaxLength(100)]
	public string SMTPFromEmailAddress { get; set; } 
 
	[MaxLength(100)]
	public string SMTPHostName { get; set; } 
 
	[MaxLength(100)]
	public string SMTPPassword { get; set; } 

	public Nullable<int> SMTPPort { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int SMTPSettingsId { get; set; } 
 
	[MaxLength(100)]
	public string SMTPUserName { get; set; } 

	public Nullable<int> StatusId { get; set; } 

	public SMTPSetting()
	{
    }

    }
}

